WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -369,14 +369,4 @@ public static boolean hasCustomDomain(@NonNull SiteModel site) {
     public static boolean hasFullAccessToContent(@Nullable SiteModel site) {
         return site != null && (site.isSelfHostedAdmin() || site.getHasCapabilityEditPages());
     }
-
-    // TODO: Inline this method when legacy MySiteFragment is removed
-    public static boolean isScanEnabled(boolean scanFeatureFlagEnabled, boolean scanPurchased, SiteModel site) {
```

Comment:
❓ I'm wondering if we should wait with removal of this method until MySiteFragment gets replaced with the ImprovedMySiteFragment. Someone might not realize both of these fragment exist and update the condition in just one of them. Wdyt?

Reply:
Good point @malinajirka. If you decide to change this @ParaskP7 , I can run through my tests again. Just let me know.

Reply:
👋 Jirka!

Yeap, I was thinking about that too, I think it is better for me to revert that change, we already talked about that when that got introduced in the codebase. I just wanted to clean this up as well, but, it is better to be on the safe side for now.

PS: @zwarm , I'll do this tomorrow and let you know. 🙏

Reply:
👋 @malinajirka @zwarm !

This is now done here: b7d33855cfb97d0fa3cf67b1c4708ab421ed80a0

Reply:
Thanks for making the change and it works as expected. 👍